source/spark_sql_migrations/spark_sql_migrations/current_state/create_current_state.py

```diff
@@ -0,0 +1,51 @@
+﻿import spark_sql_migrations.migrations.sql_file_executor as sql_file_executor
+from importlib.resources import contents
+from dependency_injector.wiring import Provide, inject
+from spark_sql_migrations.container import SparkSqlMigrationsContainer
+from spark_sql_migrations.models.configuration import Configuration
+
+
+def create_all_tables() -> None:
+    _create_all_tables()
+
+
+@inject
+def _create_all_tables(
+    config: Configuration = Provide[SparkSqlMigrationsContainer.configuration]
+) -> None:
+    """Executes schema and table scripts to create all tables"""
+    print("Creating all tables")
+    schema_scripts = _get_schema_scripts()
+    table_scripts = _get_table_scripts()
+
+    try:
+        for script in schema_scripts:
+            sql_file_executor.execute(
+                script, config.current_state_schemas_folder_path
+            )
+
+        for script in table_scripts:
+            sql_file_executor.execute(
+                script, config.current_state_tables_folder_path
+            )
+    except Exception as exception:
+        print(f"Schema migration failed with exception: {exception}")
```

Comment:
I'm wondering
- will this be logged to Azure Monitor? Or, what would it require to have it logged?
- do we want this print statement?

Reply:
I think we would like to use the `telemetry` package, when it is implemented :)

So the print statements should be replaced by logger instead.